README.rst

```diff
@@ -7,15 +7,56 @@ Requirements
 ------------
 
 * Python 2.7+, 3.3+
-* Django 1.8+
+* Django 1.8+, <1.9
```

Comment:
haha, just put 1.8 and that's it. you can add in parentheses info that 1.9+ will be added on a later date